Tests/Controller/HelperControllerTest.php

```diff
@@ -328,6 +329,20 @@ public function testappendFormFieldElementAction($validatorInterface)
 
         $twig = new \Twig_Environment($this->getMock('\Twig_LoaderInterface'));
         $twig->addExtension(new FormExtension($mockRenderer));
+
+        if (Kernel::MAJOR_VERSION >= 3 && Kernel::MINOR_VERSION >= 2) {
```

Comment:
Question: Why do you think it might be a bad idea to rely on `Kernel` to bring the version information? Maybe I'm mistaken, but those fields provide exactl the info we need. I'd rather switch to `Kernel::VERSION_ID >= 30200` though.